privacy-policy.html

```diff
@@ -0,0 +1,278 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="/src/output.css" rel="stylesheet" />
+    <link href="/src/input.css" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <title>TravelUp - Privacy Policy</title>
+  </head>
+  <body class="font-sans">
+    <header>
+      <nav class="flex items-center justify-between py-2 px-4">
+        <div>
+          <a href="/index.html">
+            <img
+              src="/assets/logo/travelup.svg"
+              alt="TravelUp logo"
+              width="169"
+              height="100"
+              class="py-2"
+            />
+          </a>
+        </div>
+        <div class="flex space-x-5">
+          <a
+            href="/index.html"
+            class="text-travel-orange text-lg font-semibold hover:text-orange-600"
+            >Home</a
+          >
+          <a
+            href="/about.html"
+            class="text-travel-orange text-lg font-semibold hover:text-orange-600"
+            >About</a
+          >
+          <a
+            href="/shop.html"
+            class="text-travel-orange text-lg font-semibold hover:text-orange-600"
+            >Shop</a
+          >
+          <a
+            href="/blog.html"
+            class="text-travel-orange text-lg font-semibold hover:text-orange-600"
+            >Blog</a
+          >
+          <a
+            href="/contact.html"
+            class="text-travel-orange text-lg font-semibold hover:text-orange-600"
+            >Contact</a
+          >
+        </div>
+      </nav>
+      <div class="relative">
+        <img
+          src="/assets/images/privacy-policy.jpg"
+          alt="Hero image"
+          class="w-full h-96 object-cover brightness-50"
+        />
+        <p
+          class="max-w-xl mx-auto absolute top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2 text-3xl text-white text-center px-4 py-2 rounded-lg z-10"
+        >
+          Here at TravelUp we care and respect your privacy
+        </p>
+      </div>
+    </header>
+
+    <section class="max-w-screen-xl mx-auto px-4">
+      <div class="max-w-3xl py-12 mx-auto">
+        <h2 class="text-black text-3xl font-bold">Privacy Policy</h2>
```

Comment:
Stick with brand colour of the chosen orange for which is text-travel-orange?

Reply:
Changed to text-travel-orange